script/scrape_ntd.py

```diff
@@ -0,0 +1,72 @@
+"""
+
```

Comment:
Might be worth a contextualizing comment -- is this mostly ready to become an Airflow operator and/or DAG task once we're ready...?